dolphinscheduler-ui/src/js/module/i18n/locale/en_US.js

```diff
@@ -211,8 +211,13 @@ export default {
   'Edit alarm group': 'Edit alarm group',
   'Create alarm group': 'Create alarm group',
   'Group Name': 'Group Name',
+  'Create Alarm Instance': 'Create Alarm Instance',
```

Comment:
In `en_US.js` and `zh_CN.js`, it would be better to keep one-to-one correspondence line by line.
For `The os tenant code cannot be all numbers` at line 195 in `en_US.js`, the line should be removed.

And then, the order of `Create Alarm Instance`, `Edit Alarm Instanc` and `Alarm instance name` should be same with `zh_CN.js`

Similarly, there is a pair of `Tenant Name` and `Queue `.

Reply:
The number of English lines has been adjusted